src/modules/px4iofirmware/px4io.c

```diff
@@ -312,11 +306,6 @@ user_start(int argc, char *argv[])
 	LED_RING(false);
 #endif
 
-	/* turn on servo power (if supported) */
-#ifdef POWER_SERVO
-	POWER_SERVO(true);
```

Comment:
Is that no longer a thing?

Reply:
We never did actually control that rail.